src/markdown/developers/component-practices.md

```diff
@@ -0,0 +1,89 @@
+---
+path: "/guides/component-practices"
+title: "Component Practices"
+intro: "A guide to writing components."
+---
+
+## Component Development Practices
+
+This is a summary of practices to follow when writing components. We explain them in the context of using design-system components, but they can be applied wherever you use React.
+
+### Use Existing Properties first
+
+Look for props that can be used to implement specific behaviour. Remember that you can use a number of styled props to change the look of NDS components.
```

Comment:
Should we define a “primitive component”? e.g when we talk about using existing props first, or composing out of smaller components, we could talk about building a purpose-built component out of primitive components. We might even be able to mention something somewhere about "if youre building your own primitive component you probably dont have to" because it should be NDs or something?